plugins/mission/mission_import_qgc_test.cpp

```diff
@@ -11,6 +11,13 @@
 #include "global_include.h"
 #include "log.h"
 
+// To locate QGroundControl plan file during Unit test.
+#if defined(WIN32) || defined(_WIN32) || defined(__WIN32)
```

Comment:
@shakthi-prashanth-m you can just use `#ifdef WINDOWS` it's what we already have in the build system.